src/shake_alert/shake_alert.go

```diff
@@ -60,16 +60,20 @@ func findR(data []float64) float64 {
 // detect outliers using the density-based algorithm
 // input: array of ints, where each index represents a time bin and value is the count for that time bin
 // output: values of outliers aka counts
-func FindOutliers(data []float64) []float64 {
+func FindOutliers(data []float64) ([]float64, []int) {
 	//result of outliers
 	var outliers []float64
+	var indexes []int
+
+	dataCopy := make([]float64, len(data))
+	copy(dataCopy, data)
```

Comment:
What's the purpose in the `copy()` here?

Reply:
There was a small bug where the findR function, would sort the actual array we were analyzing to calculate R. So we just gave it a copy instead.